raft/raftpb/raft.proto

```diff
@@ -88,8 +106,57 @@ enum ConfChangeType {
 }
 
 message ConfChange {
-	optional uint64          ID      = 1 [(gogoproto.nullable) = false];
-	optional ConfChangeType  Type    = 2 [(gogoproto.nullable) = false];
-	optional uint64          NodeID  = 3 [(gogoproto.nullable) = false];
-	optional bytes           Context = 4;
+	optional ConfChangeType  type    = 2 [(gogoproto.nullable) = false];
+	optional uint64          node_id = 3 [(gogoproto.nullable) = false, (gogoproto.customname) = "NodeID" ];
+	optional bytes           context = 4;
+
+	// NB: this is used only by etcd to thread through a unique identifier.
```

Comment:
This should ideally be removed, but I looked and it isn't trivial just because the etcd code would need to change a fair bit. I'd just leave this as is because avoiding it in V2 just means we have to duplicate the ConfChangeType enum and deal with all of the resulting incompatibilities.